tests/CUtilHelloWorld/c/main

```diff
@@ -0,0 +1,40 @@
+/*
+ * Skeleton for a C based Util using the uCLib
+ *
+ * author: Paolo Fabio Zaino
+ * purpose: showcase  and  test  basic functions of the
+ *          RISC OS uCLib.
+ *          Please Note: to compile  this file you need
+ *          the RISC OS uCLib installed and seen by the
+ *          RISC OS Filer.
+ * 
+ * Copyright by Paolo Fabio Zaino
+ * Distributed under Apache License 2.0
+ *
+ */
+
+// Include the uCLib header:
+#include "LibuCLib:uCLib.h"
+
+// include own header:
+#include "main.h"
+
+// define main function in ANSI C99 format:
+int main(int argc, char *argv[])
```

Comment:
The assembler hasn't setup a0 or a1, so argc or argv will not have the expected values. If I recall correctly, the example code (in OS? in DDE?) makes a fairly good C environment.

Reply:
not there yet, will look into argv/argc with the next test , but good to know, thx